the-collab-lab/tcl-12-smart-shopping-list
# Implement React Router and List/Add Item views

## Description
- Removed create-react-app boilerplate page.
- Created the browser router and routes for the main List and Add-Item views.
- Created placeholder components for the List and Add-Item views.
- Created a NavBar component with navigation links to the routes.
- Active links are displayed in bold.
- Installed `react-router-dom` and `@testing-library/react` dependencies.

## Related Issue
Closes #2 

## Acceptance Criteria

## Type of Changes
|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|  ✓ | :link: Update dependencies |

## Updates

### Before
<img width="800" alt="react-boilerplate" src="https://user-images.githubusercontent.com/39573970/95111707-bb5aa980-06f4-11eb-9f5c-55ee9c53618d.png">

### After
<img width="800" alt="Full-page-screenshot" src="https://user-images.githubusercontent.com/39573970/95111331-2fe11880-06f4-11eb-802a-6cf010b1b761.png">

## Testing Steps / QA Criteria
- From your terminal, pull down this branch with `git pull origin gj-tc-react-router` and check that branch out with `git checkout gj-tc-react-router`
- Then install the newly added dependencies
- Click the links at the bottom of the page to navigate to the specified route
